### PR TITLE
test: Add test for display prometheus

### PR DESCRIPTION
**Pull request by u5surf**
The conventional tests has no check the prometheus display. 
Even enough I make this PR presumptuous against him, because it is quite easy test case, but I consider it worth than nothing. Because it seems like that @SuperQ is a specialist of the prometheus, I expect that his review will be quite accuracy and will request adding more suitable cases.

### CODE REVIEW

**Comment by SuperQ** [comment]
One good test would be to pass the output of the plugin through `promtool check metrics`.

**Comment by u5surf** [comment]
@SuperQ Thanks for sharing ideas, it sounds good because it could be quite easy to use as below if response body can pipe. 
```
curl -s http://localhost/status | promtool check metrics
```

But I don't see how to collaborate with Test::Nginx which already used in tests.
I'll try to deep dive a bit in Test::Nginx or find somehow alternative things.

**Comment by u5surf** [comment]
@SuperQ Finally, I could introduce `promtool` in test to collaborate `Test::Nginx::add_response_body_check`

**Comment by u5surf** [comment]
@SuperQ Finally, I could introduce `promtool` in test to collaborate `Test::Nginx::add_response_body_check`

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -72,6 +72,12 @@ jobs:
         run: |
           sudo apt install -y cpanminus
           sudo cpanm install Test::Nginx::Socket
+      - name: 'prepare promtool'
+        run: |
+          sudo apt install -y curl wget
```
Comment (aminvakil) [inline .github/workflows/ci.yml:77]:
```suggestion
          sudo apt-get update && sudo apt-get install -y curl
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -72,6 +72,12 @@ jobs:
...
+          curl -s https://api.github.com/repos/prometheus/prometheus/releases/latest | grep browser_download_url | grep linux-amd64 | cut -d '"' -f 4 | wget -qi -
```
Comment (aminvakil) [inline .github/workflows/ci.yml:78]:
```suggestion
          curl -L $(curl -s https://api.github.com/repos/prometheus/prometheus/releases/latest | grep browser_download_url | grep linux-amd64 | cut -d '"' -f 4) -o prometheus-latest.tar.gz
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -72,6 +72,12 @@ jobs:
...
+          tar xvf prometheus*.tar.gz
```
Comment (aminvakil) [inline .github/workflows/ci.yml:79]:
```suggestion
          tar xvf prometheus-latest.tar.gz
```

**Review by aminvakil — COMMENTED** [review]
Not an issue at all, just nitpicking, so we won't install wget as well.
Also renaming latest prometheus tarball to a specific name.

**Review by SuperQ — APPROVED** [review]

**Pull request merged**